.travis.yml

```diff
@@ -90,5 +99,5 @@ jobs:
           on:
             repo: angular/angular.js
             all_branches: true
-            condition: $DEPLOY_CODE
+            condition: "$DEPLOY_CODE == true"
```

Comment:
Are you sure this works? (I would expect this to always evaluate to true).
Maybe `condition: env(DEPLOY_CODE) = true`?

Reply:
Yes, that works. https://unix.stackexchange.com/a/16110
I've just added the quote marks to make it explicit that this belongs together. The quote marks are only for yaml.

Reply:
I was referring to accessing `$DEPLOY_CODE` directly. I thought [these docs][1] applied to `condition` as well, but I now see it is only for `if`s. [These docs][2] apply to `condition`, so it is fine indeed :smiley:

[1]: https://docs.travis-ci.com/user/conditional-builds-stages-jobs#Specifying-conditions
[2]: https://docs.travis-ci.com/user/deployment/#Conditional-Releases-with-on%3A

Reply:
Yes, I was less than thrilled when I noticed that they are using a special language for the deploy conditions. I should visit them some time and chew them out for lack of consistency ... (they are from Berlin)